app/presenters/concerns/assessment_tasks/assess_against_legislation_presenter.rb

```diff
@@ -36,7 +36,7 @@ def policy_class_link
     def policy_class_tag
       tag.strong(
         I18n.t("policy_classes.#{policy_class.status}"),
-        class: "govuk-tag app-task-list__task-tag #{'govuk-tag--blue' if policy_class.complete?}"
+        class: "govuk-tag app-task-list__task-tag #{colour_class}"
       )
     end
```

Comment:
Definitely would be nice, as you've mentioned elsewhere, to start pulling these all into the components. But this PR has plenty going on, so can wait for another day ☺️